### PR TITLE
Only return the current_users participation for post interactions

### PR DESCRIPTION
**Pull request by jhass**
Nobody needs to know who else participated.

### CODE REVIEW

File: app/presenters/post_interaction_presenter.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    }
+  end
+
+  def participations
+    return @post.participations.none unless @current_user
+    @post.participations.where(author: @current_user.person)
+  end
+
+  def as_api(collection)
+    collection.includes(author: :profile).map {|element|
```
Comment (jhass) [inline app/presenters/post_interaction_presenter.rb]: This check is disabled in our config.

File: app/presenters/post_interaction_presenter.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+class PostInteractionPresenter
+  def initialize(post, current_user)
+    @post = post
+    @current_user = current_user
+  end
+
+  def as_json(_options={})
+    {
+      likes:          as_api(@post.likes),
```
Comment (jhass) [inline app/presenters/post_interaction_presenter.rb]: This check is different in our config

**Comment by Flaburgan** [comment]
Thank you!

**Comment by denschub** [comment]
Merged as ac52cef546929ed10640e148dbbb009fbe247b76, muchas gracias.

**Pull request closed without merging**